src/agreement/predicate_mention.py

```diff
@@ -13,7 +13,7 @@
 from filter_propositions import filter_verbal, filter_non_verbal
 
 
-def compute_predicate_mention_agreement(graph1, graph2):
+def compute_predicate_mention_agreement(graph1, graph2,ignore_filtering=False):

```

Comment:
Can you add documentation about the new ```ignore_filtering``` argument?
Does it have to have a default value, I generally prefer to do without these, if possible.